flink-connectors/flink-connector-jdbc/src/main/java/org/apache/flink/connector/jdbc/dialect/MySQLDialect.java

```diff
@@ -55,6 +55,15 @@ public JdbcRowConverter getRowConverter(RowType rowType) {
 		return new MySQLRowConverter(rowType);
 	}
 
+	@Override
+	public String getLimit(long limit) {
```

Comment:
how about rename to `getLimitStatement` to make the function definition more clear?